src/utils.js

```diff
@@ -492,7 +493,12 @@ const recursiveGetKeyByDepth = (event, keyToSearch, maxDepth) => {
 };
 
 export const md5Hash = item => {
-  const md5sum = crypto.createHash('md5');
-  md5sum.update(JSON.stringify(item));
-  return md5sum.digest('hex');
+  try {
```

Comment:
You can use `safeExecute`

Reply:
That's not about the "safeness" but about don't care if we failed 🤷